### PR TITLE
Add badges and stuff

### PR DESCRIPTION
**Pull request by Fryguy**
- Add CodeClimate configuration.
- Run `bundle gem` against the repo to default most files.
- Switch from coveralls to codeclimate-test-reporter
- Add the badges.

### CODE REVIEW

**Comment by Fryguy** [comment]
@bdunne Please review.  Once this is merged, we should close and reopen all of the other PRs to get them to run Travis.

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+require "manageiq/gems/pending"
```
Comment (bdunne) [inline bin/console]: This could be `require "manageiq-gems-pending"`

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-require 'vcr'
+require "simplecov"
+SimpleCov.start { command_name "spec" }
 
-if ENV["TRAVIS"]
-  require 'coveralls'
-  Coveralls.wear_merged! { add_filter("/spec/") }
-end
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'manageiq/gems/pending'
```
Comment (bdunne) [inline spec/spec_helper.rb]: same

File: test/test_helper.rb [hunk 3]
```diff
@@ -1,11 +1,7 @@
-require_relative '../lib/manageiq/gems/pending'
+require 'simplecov'
+SimpleCov.start { command_name "test" }
 
-if ENV["TRAVIS"]
-  require 'coveralls'
-  Coveralls.wear_merged! { add_filter("/test/") }
-end
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'manageiq/gems/pending'
```
Comment (bdunne) [inline test/test_helper.rb]: same

**Comment by Fryguy** [comment]
Could be, but that's the way `bundle gem` emits a require...I think I'd rather stick with the default, right?

**Pull request merged**